### PR TITLE
1529/feat/app tool bar tests

### PR DESCRIPTION
**Pull request by Noam6421**
In this PR I started writing tests for the AppToolbar component.

### CODE REVIEW

File: client/src/components/App/AppToolbar/AppToolbar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        expect((wrapper.find('p#welcome-message').props().children)?.toString()).toBe(welcomeMessage);
+    });
+
+    // it('shows toggle' , () => {
+    //     const toogleTooltipSelector = wrapper.find('#toggle-tooltip');
+    //     jest.spyOn(useAppToolbar, 'useAppToolbar').mockImplementation(() => ([user, true, jest.fn(), jest.fn()]));
+
+    //     expect(toogleTooltipSelector.exists()).toBeTruthy();
+    //     expect(wrapper.find(toogleTooltipSelector).props().title).toBe(toggleMessage);    
+    // });
```
Comment (y-golde) [inline client/src/components/App/AppToolbar/AppToolbar.test.tsx:65]: remove

File: client/src/Utils/Testing/AppToolbar/mockUseAppToolbar.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import useAppToolbar from '../../../components/App/AppToolbar/useAppToolbar';
+
+
```
Comment (y-golde) [inline client/src/Utils/Testing/AppToolbar/mockUseAppToolbar.ts:3]: remove one space

**Review by y-golde — CHANGES_REQUESTED** [review]

File: client/src/Utils/Testing/AppToolbar/mockUseAppToolbar.ts (hunk 2 above)
Comment (Noam6421) [reply client/src/Utils/Testing/AppToolbar/mockUseAppToolbar.ts:3]: removed one line

File: client/src/components/App/AppToolbar/AppToolbar.test.tsx (hunk 1 above)
Comment (Noam6421) [reply client/src/components/App/AppToolbar/AppToolbar.test.tsx:65]: removed it

**Review by y-golde — APPROVED** [review]

**Review by barnahum8 — APPROVED** [review]

**Pull request merged**